desktop/core/src/desktop/js/apps/admin/Configuration/ConfigurationTab.tsx

```diff
@@ -0,0 +1,157 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useState, useEffect, useMemo } from 'react';
+import { Spin, Alert } from 'antd';
+import { i18nReact } from '../../../utils/i18nReact';
+import './Configuration.scss';
+import { DEFAULT_SELECTED_APP, GET_CONFIG_API } from './utils';
+import AdminHeader from '../AdminHeader';
+import { ConfigurationValue } from './ConfigurationValue';
+import { ConfigurationKey } from './ConfigurationKey';
+
+interface App {
+  name: string;
+  has_ui: boolean;
+  display_name: string;
+}
+
+export interface ConfigValue {
+  help: string;
+  key: string;
+  is_anonymous: boolean;
+  values?: ConfigValue[];
+  default?: string;
+  value?: string;
+}
+
+interface Config {
+  help: string;
+  key: string;
+  is_anonymous: boolean;
+  values: ConfigValue[];
+}
+
+interface HueConfig {
+  apps: App[];
+  config: Config[];
+  conf_dir: string;
+}
+
+const Configuration: React.FC = (): JSX.Element => {
+  const { t } = i18nReact.useTranslation();
+  const [hueConfig, setHueConfig] = useState<HueConfig>();
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState<string>();
+  const [selectedApp, setSelectedApp] = useState<string>(DEFAULT_SELECTED_APP);
+  const [filter, setFilter] = useState<string>('');
+
+  useEffect(() => {
+    const fetchHueConfig = async () => {
```

Comment:
Did you look at `getLastKnownConfig` in `desktop/core/src/desktop/js/config/hueConfig.ts`?
I think it does what you are doing here

Reply:
The name is same but these 2 different configs, the content for both is different. Here we want for `ini` but the one you refer to is for `UI` configuration.

Reply:
Yup changed in the code. Thank you.